File: code/design-patterns/behavioral/command/go/command_usage.go
package main

func main() {
	turnOnCommand := &TurnOnCommand{}
	turnOffCommand := &TurnOffCommand{}
	volumeUpCommand := &VolumeUpCommand{}
	volumeDownCommand := &VolumeDownCommand{}

	onButton := NewButton(turnOnCommand)
	offButton := NewButton(turnOffCommand)
	volumeUpButton := NewButton(volumeUpCommand)
	volumeDownButton := NewButton(volumeDownCommand)

	onButton.OnClick()
	offButton.OnClick()
	volumeUpButton.OnClick()
	volumeDownButton.OnClick()
}

File: code/design-patterns/behavioral/command/python/command_usage.py
from command import (
    Button,
    TurnOnCommand,
    TurnOffCommand,
    VolumeUpCommand,
    VolumeDownCommand,
)

remote_on_button = Button(TurnOnCommand())
remote_off_button = Button(TurnOffCommand())
remote_volume_up_button = Button(VolumeUpCommand())
remote_volume_down_button = Button(VolumeDownCommand())

tv_on_button = Button(TurnOnCommand())
tv_off_button = Button(TurnOffCommand())
tv_volume_up_button = Button(VolumeUpCommand())
tv_volume_down_button = Button(VolumeDownCommand())
